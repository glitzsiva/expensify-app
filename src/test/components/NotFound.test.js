import React from 'react';
import {shallow} from 'enzyme';
import NotFound from '../../component/NotFound';

test( 'Not found Error Page',()=>{
    const wrapper =shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
});