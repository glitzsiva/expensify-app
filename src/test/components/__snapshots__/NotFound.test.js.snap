// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found Error Page 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
     Home
  </Link>
</div>
`;
